### PR TITLE
tutorial/other_packages/prebuilt_libraries

### PR DESCRIPTION
**Pull request by lasote**
Examples: https://github.com/conan-io/examples2/pull/27

### CODE REVIEW

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst [hunk 1]
```diff
@@ -1,2 +1,414 @@
...
+
+There are specific scenarios in which it is necessary to create packages from existing binaries, for example from 3rd
+parties or binaries previously built by another process or team that is not using Conan. Under these circumstances,
+building from sources is not what you want.
+
+You can package the local files in the following scenarios:
+
+ 1. When you are developing your package locally and you want to export the built artifacts to the local
+    cache. As you don't want to rebuild again (clean copy) your artifacts, you don't want to call
+    :command:`conan create`. This method will keep your build cache if you are using an IDE.
```
Comment (memsharded) [inline tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:12]:
```suggestion
 1. When you are developing your package locally and you want to quickly create a package with the built artifacts, but as you don't want to rebuild again (clean copy) your artifacts, you don't want to call
    :command:`conan create`. This method will keep your local project build if you are using an IDE.
```

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst [hunk 2]
```diff
@@ -1,2 +1,414 @@
...
+
+
+Locally building binaries
+-------------------------
+
+This is an example of scenario 1 explained in the introduction. In this case, you can run :command:`conan export-pkg`
+command directly.
+
+Please, first clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
```
Comment (memsharded) [inline tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:25]: Isn't this example reproducible from the ``conan new`` template? In theory it should, and things would be a bit simpler?

**Review by memsharded — APPROVED** [review]
Looking great

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst (hunk 2 above)
Comment (lasote) [reply tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:24]: If you change the `conanfile.py` it is.

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst [hunk 3]
```diff
@@ -1,2 +1,413 @@
...
+point to the correct places:
+
+- **os.path.join(self.source_folder, self.cpp.source.includedirs[0])** will always point to our local include folder.
+- **os.path.join(self.build_folder, self.cpp.build.libdirs[0])** will always point to the location of the libraries when
+  they are built, no matter if using a single-config CMake Generator or a multi-config one.
+
+Now, for every different configuration (different compilers, architectures, build_type...):
+
+1. We call :command:`conan install` to generate the ``conan_toolchain.cmake`` file and the ``CMakeUserPresets.json``
+   that we can be used in our IDE or calling CMake (only >= 3.23).
```
Comment (czoido) [inline tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:101]:
```suggestion
   that can be used in our IDE or calling CMake (only >= 3.23).
```

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst (hunk 2 above)
Comment (lasote) [reply tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:24]: I've changed it. Please review

**Review by czoido — APPROVED** [review]

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst [hunk 4]
```diff
@@ -1,2 +1,389 @@
...
+1. We call :command:`conan install` to generate the ``conan_toolchain.cmake`` file and the ``CMakeUserPresets.json``
+   that can be used in our IDE or calling CMake (only >= 3.23).
+
+   .. code-block:: bash
+
+       $ conan install . -s build_type=Release
+
+2. We build our project calling CMake, our IDE, ... etc:
+
+   - *Traditional way*:
```
Comment (czoido) [inline tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:61]: I woul just give one of the two alternatives, as this is not the focus of the tutorial and makes it a bit more confusing. You could choose for example the presets one and just mention that you could do it without presets or the opposite, but just having one code snippet.

File: tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst [hunk 5]
```diff
@@ -1,2 +1,389 @@
...
+    :command:`conan create`. This method will keep your local project build if you are using an IDE.
+ 2. When you cannot build the packages from sources (when only pre-built binaries are available) and you have them
+    in a local directory.
+ 3. Same as 2 but you have the precompiled libraries in a remote repository.
+
+
+Locally building binaries
+-------------------------
+
+Use the :command:`conan new` command to create a "Hello World" C++ library example project:
```
[the remark this answers is not in the document]
Comment (lasote) [reply tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:20]: Actually, it works perfectly. But it is not the best to explain what is happening IMO

[the remark this answers is not in the document]
Comment (lasote) [reply tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst:20]: Changed the explanations about the alternative/generic package method. Not sure what more Can i improve.

**Review by memsharded — APPROVED** [review]

**Pull request merged**